sdk/src/main/scala/com/horizen/block/SidechainBlockBase.scala

```diff
@@ -135,7 +138,7 @@ abstract class SidechainBlockBase[TX <: Transaction, H <: SidechainBlockHeaderBa
       throw new InvalidSidechainBlockDataException(s"${getClass.getSimpleName} $id sidechain transactions amount exceeds the limit.")
 
     // Check Block size
-    val blockSize: Int = bytes.length
+    val blockSize: Long = bytes.length
```

Comment:
why do we use `Long`.
In most of other places we use `Int`, for example: `def getMaxBlockOverheadSize() : Int`

Reply:
`EthereumTransaction size()` method returns a `Long`